lingetic-nextjs-frontend/app/languages/learn/[language]/page.tsx

```
@@ -24,19 +21,17 @@ export default function LearnPage() {
-        </button>
-      </div>
-    </QueryClientProvider>
+    <div className="container mx-auto p-4">
+      <h1 className="text-skin-base text-3xl font-bold mb-6">
+        Learning {language}
+      </h1>
+      <FillInTheBlanks question={currentQuestion} />
+      <button
+        onClick={onNext}
+        className="mt-4 bg-skin-button-primary text-skin-inverted px-4 py-2 rounded transition-colors"
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)